### PR TITLE
Windows and Linux keymap

### PR DESCRIPTION
**Pull request by gquintana**

### CODE REVIEW

File: keymaps/intellij-idea-keymap.cson [hunk 1]
```diff
@@ -82,3 +99,89 @@
...
+
+  ###########
+  # Editing #
+  ###########
+
+  # Duplicate current line or selected block
+  'ctrl-d': 'editor:duplicate-lines'
+
+  # Select successively increasing code blocks
+  'alt-up': 'editor:select-to-next-subword-boundary'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson]: Can't this be added to the multi-platform part?

File: keymaps/intellij-idea-keymap.cson [hunk 2]
```diff
@@ -82,3 +99,89 @@
...
+
+  # Decrease current selection to previous state
+  'alt-down': 'editor:select-to-previous-subword-boundary'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson]: Can't this be added to the multi-platform part?

File: keymaps/intellij-idea-keymap.cson [hunk 3]
```diff
@@ -82,3 +99,89 @@
...
+
+  # Delete line at caret
+  'shift-delete': 'editor:delete-line'
+  'ctrl-y': 'editor:delete-line'
+
+  # Smart line join
+  'ctrl-shift-j': 'editor:join-lines'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson]: Is this needed here as it is already in the multi-platform part?

File: keymaps/intellij-idea-keymap.cson [hunk 4]
```diff
@@ -82,3 +99,89 @@
...
+  'alt-left': 'pane:show-previous-item'
+
+  # Go to line
+  'ctrl-l': 'go-to-line:toggle'
+
+  # Recent files popup
+  'ctrl-e': 'fuzzy-finder:toggle-buffer-finder'
+
+  # Select current file or symbol in any view
+  'alt-f1': 'tree-view:reveal-active-file'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson:171]: Can't this be added to the multi-platform part?

File: keymaps/intellij-idea-keymap.cson [hunk 5]
```diff
@@ -82,3 +99,89 @@
...
+  ##############
+
+  # Go to class
+  'ctrl-n': 'fuzzy-finder:toggle-file-finder'
+
+  # Go to file
+  'ctrl-shift-n': 'fuzzy-finder:toggle-file-finder'
+
+  # Go to next editor tab
+  'alt-right': 'pane:show-next-item'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson:159]: Could you maybe add this to the MAC keymap as `ctrl-right` such that it is consistent?

File: keymaps/intellij-idea-keymap.cson [hunk 6]
```diff
@@ -82,3 +99,89 @@
...
+
+  # Go to previous editor tab
+  'alt-left': 'pane:show-previous-item'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson:162]: Could you maybe add this to the MAC keymap as `ctrl-left` such that it is consistent?

**Comment by guylabs** [comment]
Hi @gquintana, 

thanks a lot for your contribution! Really nice that you added the windows and linux keymaps! I just added some comments. Could you maybe check them and if you want we can discuss it before we merge it to master. After that I will create a new release such that all are able to use your keymap.

Thanks again and regards,

Guy

File: keymaps/intellij-idea-keymap.cson (hunk 6 above)
Comment (gquintana) [inline keymaps/intellij-idea-keymap.cson:162]: I don't have a Mac but reading IntelliJ's reference card, it's ctrl+right/left on Mac but alt+right/left on Win/Linux.

File: keymaps/intellij-idea-keymap.cson (hunk 3 above)
Comment (gquintana) [inline keymaps/intellij-idea-keymap.cson]: Actually, it's ctrl-w/ctrl-shift-w on Windows/Linux

**Comment by guylabs** [comment]
Hi @gquintana,

sorry for my late reply but thanks a lot for your changes! I will adapt some things and then release a new version.

Thanks and regards,

Guy

**Pull request merged**